icu4j/main/classes/core/src/com/ibm/icu/impl/locale/LocaleDistance.java

```diff
@@ -246,7 +284,7 @@ public int getBestIndexAndDistance(LSR desired, LSR[] supportedLSRs,
             if (favorSubtag == FavorSubtag.SCRIPT) {
                 distance >>= 2;
             }
-            if (distance >= threshold) {
+            if (distance >= roundedThreshold) {
```

Comment:
Is this a rounded threshold or "floor" threshold?

Reply:
it is rounded up when the fractions are not 0. see the beginning of this function.

if it was a floor and the shiftedThreshold had fraction bits, then we could ignore a language when it's really still a micro distance below the threshold.